src/components/EventManagement/EventAttendance/EventStatistics.tsx

```
@@ -0,0 +1,533 @@
+          },
+        },
+      },
+    },
+  };
+  const eventLabels = useMemo(
+    () =>
+      paginatedRecurringEvents.map((event: InterfaceEvent) => {
+        const date = (() => {
+          try {
+            const eventDate = new Date(event.startDate);
+            if (isNaN(eventDate.getTime())) {
```

Biome

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)